src/visualizer/sortingvisualizer/component/stepbystep/StepByStepUtil.js

```diff
@@ -0,0 +1,28 @@
+import SortingAlgorithmsStepByStep from '../../../algorithm/stepbysteptemplate/allSortsStepByStep';
+import { isBucketSort } from '../../util/GeneralUtil';
+
+/**
+ * Gets the step by step text to display to the user.
+ *
+ * @method
+ * @category StepTracingUtil
+ * @param {string} visualizerAlgorithm The current algorithm of the visualizer.
+ * @param {Object[]} animationArr Animation array.
```

Comment:
i think animationArr is more of any[] rather than Object[]

Reply:
Ok.